website/integrations/services/nextcloud/index.md

```diff
@@ -84,6 +87,13 @@ To set a quota set the "nextcloud_quota" property in the user's attributes. This
 If set to a value, for example `1 GB`, user(s) will have 1GB storage quota. If the attribute is not set, user(s) will have unlimited storage.
 :::
 
+:::note
+To connect to an already existing Nextcloud user, set the "nextcloud_user_id" property in the user's attributes. This must be set for individual users.
+
+The value must match the field `username` of the user on the Nextcloud instance. On Nextcloud, go to _Users_ -> check the username of the user you are trying to connect to (Under `Display name`).
```

Comment:
```suggestion
The value of `nextcloud_user_id` must match the field `username` of the user on the Nextcloud instance. On Nextcloud, go to _Users_ to see the username of the user you are trying to connect to (Under `Display name`).
```

Reply:
@Fymyte please double-check my edits to make sure I did not change the meaning, or how the UI actually appears... I wasn't sure what the arrow (->) was for?

Reply:
Thanks, the arrow was here because I thought there would be more sub-menus 😅 it had no meaning